src/Kriti/Parser.hs

```diff
@@ -74,7 +79,7 @@ instance J.FromJSON ValueExt where
 
 -- {{ range $index, $article := .event.author.articles }}
 
-type Parser = P.Parsec Void Lex.TokenStream
+type Parser = P.Parsec Lex.LexError Lex.TokenStream
```

Comment:
I used Megaparsec's custom error field to hold the LexError generated when lexing string interpolations on line151.

We could also potentially put ALL our errors in this custom type and avoid the complexity of Megaparsec's parse errors. We would have to manually throw our errors ourselves but we would have a lot more control over pretty printing and it may be simpler in the long run.